cmd/sonobuoy/app/delete.go

```diff
@@ -52,35 +53,59 @@ func NewCmdDelete() *cobra.Command {
 }
 
 func deleteSonobuoyRun(cmd *cobra.Command, args []string) {
+	s := spinner.New(spinner.CharSets[0], 100*time.Millisecond)
```

Comment:
We should const the charset, timing, and color. I'd make a helper method to create those too so that other uses (if we have them) just reference `getSpinner` for instance.

Also, nit: I prefer #14 from https://github.com/briandowns/spinner#available-character-sets :)